draft-ietf-intarea-provisioning-domains.md

```diff
@@ -233,9 +232,8 @@ some PvD Additional Information is made available through HTTP
 over TLS, as described in {{data}}.
 
 L-flag:
-: (1 bit) 'Legacy' flag stating whether
-the router is also providing IPv4 information using DHCPv4 (see
-{{dhcpv4}}).
+: (1 bit) 'Legacy' flag stating whether the PvD is associated with
+IPv4 information assigned using DHCPv4 (see {{dhcpv4}}).
```

Comment:
Indeed, it addresses Suresh's comment on this bit (no distinction by server or relay). Also more aligned with the text on DHCPv4